modules/afsocket/afsocket-source.h

```diff
@@ -52,8 +53,7 @@ struct _AFSocketSourceDriver
   LogProtoServerFactory *proto_factory;
   GSockAddr *bind_addr;
   gint max_connections;
-  gint num_connections;
-  StatsCounterItem *num_connections_stats;
+  atomic_gssize num_connections;
```

Comment:
Being an external counter, this does not need to be atomic. It can be simply gssize, and increased with `++`, and decreased with `--`, as in the original code.

Reply:
@furiel : It should be `atomic_gssize` - as the underlying api requires that.

I know that the underlying api is implemented by this PR, but I'd not change that: all the stats ctr items are atomics. There is no difference between regular and external counters from this point of view, and I'd stick to that.